ckan/logic/__init__.py

```diff
@@ -25,7 +25,14 @@ class UsernamePasswordError(Exception):
 
 
 class ActionError(Exception):
-    pass
+
+    def __init__(self, message=''):
+        self.message = message
+        super(ActionError, self).__init__(message)
+
+    def __str__(self):
```

Comment:
this should be `__unicode__` instead, for the cases where the message passed was a unicode object, right?